Adyen/Model/Installments.swift

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// The model to create an installments object.
+public struct Installments: Encodable, Equatable {
+    
+    public enum Plan: String {
+        /// Regular, monthly installment payments.
+        case regular
+        
+        /// Revolving plan, specific to some countries.
+        case revolving
+    }
+    
+    /// Selected total month of installments
+    public let totalMonths: Int
+    
+    /// Selected plan
+    public let plan: Plan
+    
+    public func encode(to encoder: Encoder) throws {
```

Comment:
this usually can have `:nodoc:` since its inherited from a documented protocol or class.